### ISSUE TITLE
Skin tenting due to fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10861]
**Preferred term label:**
Skin tenting due to fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
Sharp fragment of bone pushing against skin

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Calcaneus fractures
Clavicle shaft fracture
Humeral shaft fracture
Distal radius fracture
Radial shaft fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001384
- label: Skin tenting due to fracture
- parents: [Localized skin lesion (HP:0011355)](https://hpo.jax.org/app/browse/term/HP:0011355)
- pmid: PMID:34423090
- def: Skin tenting may occur in the setting of a fracture when a displaced bone fragment pushes on the skin, causing it to bulge outward in a tent-like shape.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10861
- synonyms:
